### ISSUE TITLE
Resources: New mining tech coming

### ISSUE THREAD

**Issue by Byrd1083** [RainbowMiner/RainbowMiner#169]
Check out some new mining tech coming to market.

- https://store.mineority.io

**Comment by RainbowMiner** [comment]
Yes, these are either supportive FPGAs, or full FPGA miner (Field Programmable Gate Array).
Those are being programmed using Verilog or similar and are being controlled by a small, operating system dependant miner programm. I am already looking into this for two reasons: first, I will integrate the miners into RainbowMiner, second, I will try to create my own miner program for those FPGA.

**Comment by jch9678** [comment]
Have you had a chance to contact SQRL Research about integrating their miner into RainbowMiner?  The software should be released this week.  You can ask for access to the api's at the bottom of this page..
http://squirrelsresearch.com/sqrl-miner/

**Comment by RainbowMiner** [comment]
I did already contact them - waiting for the API information. As soon as I get this info, I will implmement a miner module.
